pylint/extensions/check_docstring.py

```diff
@@ -0,0 +1,65 @@
+from pylint.checkers.base import BaseChecker
+from pylint.interfaces import IAstroidChecker, HIGH
+from pylint.checkers.utils import check_messages
+
+
+class _BasicChecker(BaseChecker):
```

Comment:
Just use the BaseChecker class as a base class for DocStringChecker and give it a different name than basic.